docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/ConvertTypeMultiColumn.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    // This example illustrates how to convert multiple columns of different types to one type, in this case System.Single. 
+    // This is often a useful data transformation before concatenting the features together and passing them to a particular estimator.
+    public static class ConvertTypeMultiColumn
+    {
+        // The initial data type
+        private class InputData
```

Comment:
Other places we have C# structures defined after the actual example function.

Reply:
i like that! focus should be on the example. 

---
In reply to: [271401627](https://github.com/dotnet/machinelearning/pull/3167#discussion_r271401627) [](ancestors = 271401627)